conda_smithy/templates/travis.yml.tmpl

```
@@ -47,8 +47,8 @@ script:
   - export UPLOAD_ON_BRANCH="{{ upload_on_branch }}"
 {%- endif %}
 {% if 'osx' in platformset %}
-  - if [[ ${PLATFORM} =~ .*osx.* ]]; then ./.scripts/run_osx_build.sh; fi
+  - if [[ ${PLATFORM} =~ .*osx.* ]]; then {% if idle_timeout_minutes %}travis_wait {{ idle_timeout_minutes }}{% endif %} ./.scripts/run_osx_build.sh; fi
 {%- endif %}
 {% if 'linux' in platformset %}
-  - if [[ ${PLATFORM} =~ .*linux.* ]]; then ./.scripts/run_docker_build.sh; fi
-{%- endif %}
+  - if [[ ${PLATFORM} =~ .*linux.* ]]; then {% if idle_timeout_minutes %}travis_wait {{ idle_timeout_minutes }}{% endif %} ./.scripts/run_docker_build.sh; fi
```

Have you tried rerendering with this change?
Should it be,
```suggestion
  - if [[ ${PLATFORM} =~ .*linux.* ]]; then {%- if idle_timeout_minutes %}travis_wait {{ idle_timeout_minutes }}{% endif %} ./.scripts/run_docker_build.sh; fi
```

---

From a quick test it to support the whitespace stripping for both scenario's I've added the dash & an additional space before travis _wait